### ISSUE TITLE
hdr10 and hdr10+ optimization

### ISSUE THREAD

**Issue by renvan100** [rigaya/NVEnc#485]
Is it possible to set the hdr10-opt and hdr10+ optimization flags?

**Comment by rigaya** [comment]
No, unfortunately NVEnc does not support hdr10-opt or hdr10+ optimization flags.
